Source/Scene/Cesium3DTileset.js

```diff
@@ -320,116 +321,6 @@ define([
-         * </p>
-         *
-         * @type {Boolean}
-         * @default false
-         */
-        this.debugWireframe = defaultValue(options.debugWireframe, false);
-
-        /**
-         * This property is for debugging only; it is not optimized for production use.
-         * <p>
-         * When true, renders the bounding volume for each visible tile.  The bounding volume is
-         * white if the tile's content has an explicit bounding volume; otherwise, it
-         * is red.  Tiles that are not at final resolution are yellow.
-         * </p>
-         *
-         * @type {Boolean}
-         * @default false
-         */
-        this.debugShowBoundingVolume = defaultValue(options.debugShowBoundingVolume, false);
-
-        /**
-         * This property is for debugging only; it is not optimized for production use.
-         * <p>
-         * When true, renders a blue bounding volume for each tile's content.
-         * </p>
-         *
-         * @type {Boolean}
-         * @default false
-         */
-        this.debugShowContentBoundingVolume = defaultValue(options.debugShowContentBoundingVolume, false);
-
-        /**
-         * This property is for debugging only; it is not optimized for production use.
-         * <p>
-         * When true, renders the viewer request volume for each tile.
-         * </p>
-         *
-         * @type {Boolean}
-         * @default false
-         */
-        this.debugShowViewerRequestVolume = defaultValue(options.debugShowViewerRequestVolume, false);
-
-        /**
-         * This property is for debugging only; it is not optimized for production use.
-         * <p>
-         * When true, draws labels to indicate the geometric error of each tile.
-         * </p>
-         *
-         * @type {Boolean}
-         * @default false
-         */
-        this.debugShowGeometricError = defaultValue(options.debugShowGeometricError, false);
-
-        this._tileDebugLabels = undefined;
-        this.debugPickedTileLabelOnly = false;
-        this.debugPickedTile = undefined;
-        this.debugPickPosition = undefined;
-
-        /**
-         * This property is for debugging only; it is not optimized for production use.
-         * <p>
-         * When true, draws labels to indicate the number of commands, points, triangles and features of each tile.
-         * </p>
-         *
-         * @type {Boolean}
-         * @default false
-         */
-        this.debugShowRenderingStatistics = defaultValue(options.debugShowRenderingStatistics, false);
-
-        /**
-         * This property is for debugging only; it is not optimized for production use.
-         * <p>
-         * When true, draws labels to indicate the geometry and texture memory usage of each tile.
-         * </p>
-         *
-         * @type {Boolean}
-         * @default false
-         */
-        this.debugShowMemoryUsage = defaultValue(options.debugShowMemoryUsage, false);
-
-        /**
```

Comment:
Maybe due to a bad merge all these properties had been duplicated, but are removed now.